examples/ElmWithRoc/README.md

```diff
@@ -0,0 +1,34 @@
+# Building a Roc Web Server with an Elm Frontend
+
+
```

Comment:
We should include a brief description/explanation of this example here. 

Also I think we should include a link to basic-web server platform.

Why would someone want to build a web app using Roc on the back-end and Elm on the front-end?